src/javascript/app/pages/user/account/settings/iphistory/iphistory.data.js

```diff
@@ -15,8 +15,8 @@ const IPHistoryData = (() => {
             { name: 'IE',         regex: /trident\/\d+\.\d+;.*[rv:]+(\d+\.\d)/i },
             { name: 'Firefox',    regex: /firefox\/([\d\w.-]+)/i },
             { name: 'Binary app', regex: /binary\.com V([\d.]+)/i },
-            { name: 'iPhone', regex: /\b(iPhone\d*)\b.*(DP2P)\b/ig },
-            { name: 'Android', regex: /\b(Android\d*)\b.*(DP2P)\b/ig },
+            { name: 'iPhone', regex: /\b(iPhone\d*)\b.*(Deriv GO|Deriv P2P)\b/ig },
+            { name: 'Android', regex: /\b(Android\d*)\b.*(Deriv GO|Deriv P2P)\b/ig },
```

Comment:
updating the regex to match Deriv Go logout environment.
Sample environment :
environment: "12-Apr-22 09:14:59GMT IP=45.120.201.159 IP_COUNTRY=AE User_AGENT=Mozilla/5.0 (Linux; U; Android 12; SM-S908E Build/SP1A.210812.016) Deriv GO/0.8.5+8 LANG=EN"

We are getting undefined for DP2P logout. It is because Deriv P2P logout environment have Deriv P2P and not DP2P -
Sample environment :
_environment: "31-Mar-22 06:44:16GMT IP=91.75.6.225 IP_COUNTRY=AE User_AGENT=Mozilla/5.0 (Linux; U; Android 10; Mi A2 Build/QKQ1.190910.002) Deriv P2P/1.10.1+12027 LANG=EN"_